mod/01_metrics/spec/set/metric_type/formula_spec.rb

```diff
@@ -229,14 +234,16 @@ def update_formula new_formula
                      type_id: Card::PlainTextID,
                      content: build_formula("{{%s}}*5+{{%s}}*2")
       end
-      expect(calc_value).to eq("60.0")
-      expect(calc_value "Samsung", "2015").to eq("29.0")
-      expect(calc_value "Sony_Corporation").to eq("9.0")
-      expect(calc_value_card "Death_Star", "1977").to be_falsey
+      test_calculation [], "60.0"
+      test_calculation %w(Samsung 2015), "29.0"
+      test_calculation %w(Sony_Corporation), "9.0"
+      not_researched_card = calc_value_card "Death_Star", "1977"
+      expect(not_researched_card).to be_falsey
     end
   end
 
   it "handles wolfram formula" do
+    pending "restoring access to Wolfram API"
```

Comment:
note: current behavior of pending is that test code is still executed, so FYI we're still making API calls.  Hmm.  Maybe I'll comment these out...